Makefile

```diff
@@ -19,15 +19,32 @@ test_stress:
 test_reconnect:
 	go test -tags=reconnect ./...
 
+test_exactlyonce:
```

Comment:
Can probably control this via env var instead and remove this from makefile.  Then could eg stress test with exactly once configs if you wanted too.

Reply:
I guess that we can add a separate file here: https://github.com/AlexCuse/watermill-nats/tree/master/.github/workflows with that test (as we don't want to modify tests for everything).

What do you think, @m110?

Reply:
If it needs to be a separate build tag then yeah, I think it would be best to create another workflow in the repository. Alternatively, make it a make dependency of `test`?